pepys_import/core/store/data_store.py

```diff
@@ -10,6 +10,7 @@
 from sqlalchemy.exc import ArgumentError, OperationalError
 from sqlalchemy.orm import sessionmaker, undefer
 from sqlalchemy.sql import func
+from sqlalchemy_utils import merge_references
```

Comment:
Hey @BarisSari - `merge_references` is a great find.  That's doing a lot of the heavy lifting for this PR.

Reply:
Yes, @IanMayo. It finds all dependent objects and updates them.